ngraph_bridge/ngraph_tensor_manager.cc

```diff
@@ -63,6 +63,33 @@ void NGraphTensorManager::Initialize() {
       FindComplement(m_number_of_inputs, m_input_indexes_from_variables);
   m_pipelined_output_indexes =
       FindComplement(m_number_of_outputs, m_output_indexes_assigning_variable);
+
+  string node_prefetch_key =
+      NGraphCatalog::CreateNodeKey(m_ng_encap_graph_id, m_ng_encap_node_name);
+
+  if (NGraphCatalog::ExistsInPrefetchedInputIndexMap(node_prefetch_key)) {
+    auto prefetch_indexes =
+        NGraphCatalog::GetIndexesFromPrefetchedInputIndexMap(
```

Comment:
The NGraphCatalog::GetIndexesFromPrefetchedInputIndexMap function signature can be changed to accept the node key rather than creating itself to avoid mistakes and also we have the key already created.

Reply:
changed this, not creating a key